src/services/mongoose/schema.ts

```diff
@@ -408,7 +408,7 @@ UserSchema.virtual("userIsActive").get(async function (this: typeof UserSchema)
 })
 
 UserSchema.statics.getUserByPhone = async function (phone: string) {
```

Comment:
This should be moved to `UsersRepository` - its only used in admin panel and would be a quick change - but can be done in another PR.

Reply:
keeping that for another PR